### ISSUE TITLE
Technical Spike: Providing extensibility to interactivity API blocks in a new add to cart block.

### ISSUE THREAD

**Issue by samueljseay** [woothemes/woocommerce#50014]
We want to explore - at this stage in a purely experimental form - an approach to providing extensibility from new interactivity API powered blocks that WooCommerce would provide.

Specifically we're interested in what a new "add to cart" block (series of blocks most likely) would look like if it was powered by interactivity API and at what level would we expose the block for extension by developers.

The task here is to create a minimal example of one or more of the potential blocks we'd create as part of the add to cart blockification. The technical spike would be powered by interactivity API and would provide some extension points that would showcase ways a developer might integrate with this new block.

**Comment by Aljullu** [comment]
@samueljseay what do you think about closing this issue in favor of this epic: https://github.com/woocommerce/woocommerce/issues/53014? Or is there anything from this issue that you think isn't covered in the project?

**Comment by samueljseay** [comment]
@Aljullu Yep you're right this is out of date now and your epic covers it better :)
